app/controllers/qualifications/users/omniauth_callbacks_controller.rb

```diff
@@ -8,7 +8,7 @@ def identity
         @user = User.from_identity(auth)
         session[:identity_user_id] = @user.id
         session[:identity_user_token] = auth.credentials.token
-        session[:identity_user_token_expiry] = auth.credentials.expires_at
+        session[:identity_user_token_expiry] = auth.credentials.expires_in.seconds.from_now.to_i
```

Comment:
Testing this locally (using preprod Identity), I still get expires_at back, leading to an exception at this line. Any idea why?